status-im/status-desktop
# fix(@desktop/chat): Fix showing contact names after restoring account

Fixes #7241

### What does the PR do

Member names (3-word aliases) are shown correctly after restoring account from seed phrase.

**How to test:**
1. Make a backup of your Status folder
2. Remove the Status folder, run the app and restore your account.
3. Your contacts and community members should have names

### Affected areas

Member lists

### Screenshot of functionality (including design for comparison)

![image](https://user-images.githubusercontent.com/61889657/189100778-4cacc331-719e-4fa1-b59f-b74441edc294.png)

🍺
